build.gradle

```diff
@@ -244,14 +244,20 @@ task dockerCopyFiles(type: Copy) {
 task dockerBuildImage(type: DockerBuildImage) {
     dependsOn 'dockerCopyFiles'
     inputDir = project.file('build/docker/')
-    tag = "${project.dockerTagBase}/${jar.baseName}:${project.version}"
+    tags.add("${project.dockerTagBase}/${jar.baseName}:${project.version}".toString())
+    tags.add("${project.dockerTagBase}/${jar.baseName}:latest".toString())
 }
 
 task dockerPushImage(type: DockerPushImage) {
     dependsOn 'dockerBuildImage'
     imageName = "${project.dockerTagBase}/${jar.baseName}:${project.version}"
 }
 
+task dockerPushLatest(type: DockerPushImage) {
+    dependsOn 'dockerBuildImage'
+    imageName = "${project.dockerTagBase}/${jar.baseName}:latest"
+}
```

Comment:
This pushes the `:latest` image to docker-hub.  Ideally our CI process would do this IFF the branch name is `develop`  Something like a command-line build step that conditionally runs `./gradlew dockerPushLatest`